cordova-sdk/plugin.xml

```diff
@@ -1,32 +1,35 @@
 <?xml version='1.0' encoding='utf-8'?>
-<plugin id="optimove-cordova-sdk" version="2.1.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
+<plugin id="optimove-cordova-sdk" version="2.2.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
     <name>OptimoveSDKPlugin</name>
     <js-module name="OptimoveCore" src="www/OptimoveCore.js"/>
     <js-module name="Optimove" src="www/Optimove.js">
         <clobbers target="window.Optimove" />
     </js-module>
     <hook type="before_prepare" src="scripts/inject-config.js" />
+
     <platform name="ios">
-        <resource-file src="scripts/optimove.plist" target="optimove.plist" />
-        <config-file target="config.xml" parent="/*">
+        <config-file parent="/*" target="config.xml">
             <feature name="OptimoveSDKPlugin">
                 <param name="ios-package" value="Optimove_Cordova" />
             </feature>
+            <preference name="SwiftVersion" value="5" />
```

Comment:
if by default it doesnt set swift version of the project. Does it mean setting it to 5 is what potentially could lead to conflicts? (especially if 6 is released 1 day)

Reply:
Cordova could operate with Swift version 4. It is good practice to set the language version.
When it will be Swift 6 -- we will change it here as well

Reply:
would this force consumers of the plugin into swift 6 as well?

Reply:
> would this force consumers of the plugin into swift 6 as well?

Once we will stop support Swift 5, it is highly possible that it will force them to use Swift 6.

Reply:
still dont undertsand, what happens if we set swift 6, other plugin sets swift 5? plugin added later wins?

Reply:
The build does not pass CI if delete this line:
https://github.com/optimove-tech/Optimove-SDK-Cordova/actions/runs/7195249630/job/19597497704#step:6:71

Reply:
> still dont undertsand, what happens if we set swift 6, other plugin sets swift 5? plugin added later wins?

We will see it in the future.